site/_docs/reference.md

```diff
@@ -2097,6 +2097,7 @@ semantics.
 | m | JSON_LENGTH(jsonValue [, path ])               | Returns a integer indicating the length of *jsonValue*
 | m | JSON_KEYS(jsonValue [, path ])                 | Returns a string indicating the keys of a JSON *jsonValue*
 | m | JSON_REMOVE(jsonValue, path[, path])           | Removes data from *jsonValue* using a series of *path* expressions and returns the result
+| m | JSON_STORAGE_SIZE(jsonValue)                   | Returns an integer value indicating the size of a *jsonValue*
```

Comment:
"size" is too simple to describe the return. You can just use the words that MySQL already used, or make it anyway more detailed.